### PR TITLE
Layout, type and headings

### PR DESCRIPTION
**Pull request by zaccolley**
Homepage:
homepag

Homepage mobile:
homepage-mobile

### CODE REVIEW

**Comment by zaccolley** [comment]
Will try get this to pass for you tomorrow Kim :+1:

**Comment by zaccolley** [comment]
Have made headings consistent on most pages, and added some where it makes sense. Will update the original post with screenshots.

**Comment by zaccolley** [comment]
I've also moved the 'Blog' link to the right and moved the 'Menu' icon to the right also.

**Comment by zaccolley** [comment]
This starts to cover some of the headings mentioned in #582

**Comment by zaccolley** [comment]
Covers some colour accessibility issues around body content in #583

**Comment by zaccolley** [comment]
hey @KimberleyCook can I get a review on this please? :+1:

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -131,34 +134,23 @@ u {
   padding-bottom: 10px;
 }
 
-.stripe:not(.reverse):first-child {
-  padding: 80px 0px 40px;
+.stripe {
+  padding: 20px 0;
 }
```
Comment (KimberleyCook) [inline app/assets/stylesheets/main.scss]: How come you removed these paddings?

For me the screenshots you've shown feel a little too squashed together now.

File: app/views/invitation/show.html.haml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .stripe.reverse
   .row
-    .large-12.columns
-      %h2
+    .columns
+      %h1
```
Comment (KimberleyCook) [inline app/views/invitation/show.html.haml]: in the world of foundation this `large-12` class should still be there right?

File: app/views/shared/_title.html.haml [hunk 3]
```diff
@@ -1,6 +1,6 @@
-%section.hero.opaque
+%section.stripe.reverse
   .row
```
Comment (KimberleyCook) [inline app/views/shared/_title.html.haml]: I haven't been able to double check the css so you may need to ignore me, but these deleted id's don't have any related styles that need deleting do they?

File: app/views/workshops/_workshop_actions.html.haml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-%p
+%p.h4
```
Comment (KimberleyCook) [inline app/views/workshops/_workshop_actions.html.haml]: is this right? Wouldn't this mean it's a paragraph with a `h4` class attached?

**Comment by zaccolley** [comment]
Thanks for this, will get to this on Tuesday!

File: app/views/invitation/show.html.haml (hunk 2 above)
Comment (zaccolley) [reply app/views/invitation/show.html.haml]: yep you're right, will add that in, thanks!

File: app/views/workshops/_workshop_actions.html.haml (hunk 4 above)
Comment (zaccolley) [reply app/views/workshops/_workshop_actions.html.haml]: These helper classes aren't in the Foundation we're using but the latest version does. The docs are here: http://foundation.zurb.com/sites/docs/typography-helpers.html#typescale

I've added them in to our version, it's so we can have it presentationally look the same as a h4 element but the markup is still a paragraph.

Might be a better way of styling that but I think the classes make some sense, "this should look the same as a h4"?

File: app/assets/stylesheets/main.scss (hunk 1 above)
Comment (zaccolley) [reply app/assets/stylesheets/main.scss]: I removed these for mobile view, but I'll tweak them cause I agree

File: app/views/shared/_title.html.haml (hunk 3 above)
Comment (zaccolley) [reply app/views/shared/_title.html.haml]: I have fixed this in e95457f below

File: app/assets/stylesheets/main.scss (hunk 1 above)
Comment (zaccolley) [reply app/assets/stylesheets/main.scss]: OK I have tweaked the spacing, and the type a bit more, you can see how that looks in the updated screenshots in the original comment on this PR.

**Comment by zaccolley** [comment]
I've restructured the homepage a bit more. Have created a new 'Support codebar' section at the bottom as it seems the 'Sign up for codebar' is important to keep clearer.

What do you think? @KimberleyCook @matyikriszta

**Comment by zaccolley** [comment]
closing cause i dont want to have to keep this synced with master

**Pull request closed without merging**